live_pipeline/transform/conftest.py

```diff
@@ -0,0 +1,111 @@
+import pytest
+import pandas as pd
+
+
+@pytest.fixture
+def fake_raw_plant_data():
+    return {
+        "botanist": {
+            "email": "fake_person@lnhm.co.uk",
+            "name": "Fake Person",
+            "phone": "001-481-273-3691x134"
+        },
+        "last_watered": "Wed, 30 Aug 2023 13:54:32 GMT",
+        "name": "Fake plant",
+        "origin_location": [
+            "7.55",
+            "-118.03917",
+            "Fake location",
+            "Fake",
+            "Continent/Fake"
+        ],
+        "plant_id": 532,
+        "recording_taken": "2023-08-30 14:56:09",
+        "soil_moisture": 96.36114621062993,
+        "temperature": "12.070482937725064"
+    }
+
+
+@pytest.fixture
+def fake_data():
+    return {
+        "1": {
+            "botanist": {
+                "email": "fake_person@lnhm.co.uk",
+                "name": "Fake Person",
+                "phone": "001-481-273-3691x134"
+            },
+            "last_watered": "Wed, 30 Aug 2023 13:54:32 GMT",
+            "name": "Fake plant",
+            "origin_location": [
+                "7.55",
+                "-118.03917",
+                "Fake location",
+                "Fake",
+                "Continent/Fake"
+            ],
+            "plant_id": 532,
+            "recording_taken": "2023-08-30 14:56:09",
+            "soil_moisture": 96.36114621062993,
+            "temperature": "12.070482937725064"
+        },
+        "2": {
+            "botanist": {
+                "email": "fake_person@lnhm.co.uk",
+                "name": "Fake Person",
+                "phone": "001-481-273-3691x134"
+            },
+            "last_watered": "Wed, 30 Aug 2023 13:54:32 GMT",
+            "name": "Fake plant",
+            "origin_location": [
+                "7.55",
+                "-118.03917",
+                "Fake location",
+                "Fake",
+                "Continent/Fake"
+            ],
+            "plant_id": 532,
+            "recording_taken": "2023-08-30 14:56:09",
+            "soil_moisture": 96.36114621062993,
+            "temperature": "12.070482937725064"
+        }}
+
+
+@pytest.fixture
+def fake_time_recorded():
+    return "2023-08-30 14:56:09"
+
+
+@pytest.fixture
+def fake_time_recorded_invalid():
+    return "2023-13-30 14:56:09"
+
+
+@pytest.fixture
+def fake_last_watered():
+    return "Wed, 30 Aug 2023 14:10:54 GMT"
+
+
+@pytest.fixture
+def fake_temperature_valid():
+    return 25.3252
+
+
+@pytest.fixture
+def fake_temperature_invalid():
+    return 100.32536
+
+
+@pytest.fixture
+def fake_soil_moisture_valid():
+    return 80.000352
```

Comment:
Great coverage of edge cases and I love the use of the `conftest` file to define these.